qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -86,6 +86,8 @@
             no Windows
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "rm -f /tmp/foo; echo foo > /tmp/foo"
+        - check_reset_reboot_shutdown_fsfreeze:
```

Comment:
Should the isa_serial.check_reset_reboot_shutdown_fsfreeze need to be supported?